source/features/rgh-feature-descriptions.tsx

```diff
@@ -22,14 +22,18 @@ async function add(infoBanner: HTMLElement): Promise<void> {
 	}
 
 	const conversationsUrl = new URL('https://github.com/refined-github/refined-github/issues');
-	conversationsUrl.searchParams.set('q', `sort:updated-desc "${feature.id}"`);
+	conversationsUrl.searchParams.set('q', `sort:updated-desc is:open "${feature.id}"`);
```

Comment:
Change: only show open issues/PRs by default. Seeing current bugs is more useful than "every discussion"